pkg/api/mlflow/controller/artifact.go

```diff
@@ -26,3 +33,47 @@ func (c Controller) ListArtifacts(ctx *fiber.Ctx) error {
 	log.Debugf("artifactList response: %#v", resp)
 	return ctx.JSON(resp)
 }
+
+// GetArtifact handles `GET /artifacts/get` endpoint.
+func (c Controller) GetArtifact(ctx *fiber.Ctx) error {
+	req := request.GetArtifactRequest{}
+	if err := ctx.QueryParser(&req); err != nil {
+		return api.NewBadRequestError(err.Error())
+	}
+	log.Debugf("GetArtifact request: %#v", req)
+
+	artifact, err := c.artifactService.GetArtifact(ctx.Context(), &req)
+	if err != nil {
+		return err
+	}
+
+	filename := filepath.Base(req.Path)
+	ctx.Set("Content-Type", common.GetContentType(filename))
+	ctx.Set("Content-Disposition", fmt.Sprintf("attachment; filename=%s", filename))
+	ctx.Set("X-Content-Type-Options", "nosniff")
+	ctx.Context().Response.SetBodyStreamWriter(func(w *bufio.Writer) {
+		defer artifact.Close()
+
+		start := time.Now()
+		if err := func() error {
+			bytesWritten, err := io.CopyBuffer(w, artifact, make([]byte, 4096))
+			if err != nil {
+				return fmt.Errorf("unable to copy artifact Reader to output stream: %w", err)
+			}
+			if err := w.Flush(); err != nil {
+				return fmt.Errorf("unable to flush output stream: %w", err)
+			}
+			log.Debugf("GetArtifact wrote bytes to output stream: %d", bytesWritten)
+			return nil
+		}(); err != nil {
+			log.Errorf(
+				"error encountered in %s %s: error streaming artifact: %s",
+				ctx.Method(),
+				ctx.Path(),
+				err,
+			)
+		}
+		log.Infof("body - %s %s %s", time.Since(start), ctx.Method(), ctx.Path())
```

Comment:
can you please add a bit more information here like:
```
time spent: %s, method: %s, route: %s
```

Reply:
this was following example @jgiannuzzi requested